server/routes/auth.routes.js

```diff
@@ -162,89 +154,30 @@ function createUser(loginGovUser) {
-  },
-  'Department of Health and Human Services': {
-    'National Institutes of Health': ['NIH', 'NATIONAL INSTITUTES OF HEALTH'],
-    'Food and Drug Administration': ['FDA', 'FOOD AND DRUG ADMINISTRATION'],
-    'Indian Health Service': ['IHS', 'INDIAN HEALTH SERVICE'],
-    'Centers for Medicare & Medicaid Services': ['CMS', 'CENTER FOR MEDICARE AND MEDICAID SERVICES']
-  },
-  'Department of Homeland Security': {
-    'Federal Emergency Management Agency': ['FEMA'],
-    'U.S. Citizenship and Immigration Services': ['USCIS'],
-    'U.S. Secret Service': ['USSS', 'SECRET SERVICE']
-  },
-  'Department of Commerce': {
-    'National Oceanic and Atmospheric Administration': ['NOAA'],
-    'National Telecommunications and Information Administration': ['NTIA']
-  },
-  'Department of the Interior': {
-    'National Park Service': ['NPS'],
-    'Fish and Wildlife Service': ['FWS', 'FISH AND WILDLIFE'],
-    'Bureau of Ocean Energy Management': ['BOEM']
-  },
-  'Department of the Treasury': {
-    'Internal Revenue Service': ['IRS'],
-    'U.S. Mint': ['MINT', 'US MINT']
-  }
-};
-
-
 function getOfficeFromEmail(email) {
+  if (!email) return null;
   
-  if (!email) {
-    return null;
-  }
-
   const emailLower = email.toLowerCase();
-  // DOD Mappings
-  if (emailLower.includes('army.mil')) return 'DEPT OF THE ARMY';
-  if (emailLower.includes('navy.mil') || emailLower.includes('us.navy.mil')) return 'DEPT OF THE NAVY';
-  if (emailLower.includes('af.mil') || emailLower.includes('us.af.mil')) return 'DEPT OF THE AIR FORCE';
-  if (emailLower.includes('spaceforce.mil')) return 'Space Force';
-  if (emailLower.includes('dla.mil')) return 'DEFENSE LOGISTICS AGENCY';
- 
-  // HHS Mappings
-  if (emailLower.includes('nih.gov')) return 'NATIONAL INSTITUTES OF HEALTH';
-  if (emailLower.includes('fda.hhs.gov')) return 'FOOD AND DRUG ADMINISTRATION';
-  if (emailLower.includes('ihs.gov')) return 'INDIAN HEALTH SERVICE';
-  if (emailLower.includes('cms.hhs.gov')) return 'CENTERS FOR MEDICARE AND MEDICAID SERVICES';
- 
-  // DHS Mappings
-  if (emailLower.includes('fema.dhs.gov')) return 'FEDERAL EMERGENCY MANAGEMENT AGENCY';
-  if (emailLower.includes('uscis.dhs.gov')) return 'US CITIZENSHIP AND IMMIGRATION SERVICES';
-  if (emailLower.includes('usss.dhs.gov')) return 'US SECRET SERVICE';
- 
-  // Commerce Mappings
-  if (emailLower.includes('noaa.gov')) return 'NATIONAL OCEANIC AND ATMOSPHERIC ADMINISTRATION';
-  if (emailLower.includes('ntia')) return 'NATIONAL TELECOMMUNICATIONS AND INFORMATION ADMINISTRATION';
- 
-  // Interior Mappings
-  if (emailLower.includes('nps.gov')) return 'NATIONAL PARK SERVICE';
-  if (emailLower.includes('fws.gov')) return 'US FISH AND WILDLIFE SERVICE';
-  if (emailLower.includes('boem.gov')) return 'BUREAU OF OCEAN ENERGY MANAGEMENT';
- 
-  // Treasury Mappings
-  if (emailLower.includes('irs.gov')) return 'INTERNAL REVENUE SERVICE';
-  if (emailLower.includes('usmint.treas.gov')) return 'US MINT';
-
```

Comment:
I would like all the stuff above added to the config.js under the AGENCY_LOOKUP if the acronyms are not already there.